addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -2030,7 +2030,155 @@ export class OdooEditor extends EventTarget {
         if (direction === 'col') {
             this.editable.classList.add('o_col_resize');
         } else if (direction === 'row') {
-            this.editable.classList.add('o_resizing_row');
+            this.editable.classList.add('o_row_resize');
+        }
+    }
+    /**
+     * Setup the resizing of a table, in a given direction, "pulling" the border
+     * between the given targets (ordered left to right or top to bottom).
+     *
+     * @private
+     * @param {'col'|'row'} direction
+     * @param {HTMLTableRowElement|HTMLTableCellElement|undefined} target1
+     * @param {HTMLTableRowElement|HTMLTableCellElement|undefined} target2
+     */
+    _resizeTable(direction, target1, target2) {
+        this._isResizingTable = true;
+        this._toggleTableResizeCursor(direction);
+        const onMouseMove = ev => {
+            ev.preventDefault();
+            this._handleTableResize(ev, direction, target1, target2);
+        };
+        const stop = ev => {
+            ev.preventDefault();
+            this._isResizingTable = false;
+            this._toggleTableResizeCursor(false);
+            this.document.removeEventListener('mousemove', onMouseMove);
+            this.document.removeEventListener('mouseup', stop);
+            this.document.removeEventListener('mouseleave', stop);
+        };
+        this.document.addEventListener('mousemove', onMouseMove);
+        this.document.addEventListener('mouseup', stop);
+        this.document.addEventListener('mouseleave', stop);
+    }
+    /**
+     * Resizes a table in the given direction, by "pulling" the border between
+     * the given targets (ordered left to right or top to bottom).
+     *
+     * @private
+     * @param {MouseEvent} ev
+     */
+    _handleTableResize(ev, direction, target1, target2) {
```

Comment:
It is not obvious the difference between a function called "_resizeTable" and another called "_handleTableResize". Both look like they resize a table. I was going to suggest to rename `_resizeTable` to `_initTableResizing` then I realized it was only called once so I would just inline it at call site and put a comment that says "Initialize table resizing." on top of the relevant block of code. (Then I'd rename `_handleTableResize` to `_resizeTable` because I don't really see the added value of having "handle" in the function name, moreover, the docstring of the function starts with "resizes a table" so `_resizeTable` feels like a better name to me.)

Reply:
I made a commit that puts both together in `_onDocumentMousedown` since they're both only used once. But I think since `_handleTableResize` is very long, it might be best to keep it separate anyway and call it `_resizeTable`. But them being all together has the advantage of limiting redirections, everything is concentrated. I'll let you be the judge.